edx_repo_tools/release/tag_release.py

```diff
@@ -11,7 +11,7 @@
 5. Show the identified repos and commits, and ask for confirmation from the user
 6. Upon confirmation, create Git tags for the repos using the GitHub API
 """
-from __future__ import unicode_literals, print_function
+from __future__ import print_function
```

Comment:
BTW, I looked into what click is doing: it walks the stack looking for the future import. Ick.  But it also provides `click.disable_unicode_literals_warning` if you want to set it to shut it up.